package.json

```diff
@@ -4,11 +4,16 @@
   "description": "Flux Standard Action utlities for Redux",
   "main": "lib/index.js",
   "scripts": {
-    "test": "make test",
-    "prepublish": "make clean build"
+    "build": "npm run clean && babel src --out-dir lib --ignore *-test.js",
```

Comment:
Ah, I thought `__tests__/init.js` still existed, but I see it's been moved to `test/init.js`.

What do you think about moving the contents of `__tests__` to `test`, to consolidate (seems like test-related files are in two places right now)? This would simplify the build script to

```js
"build": "npm run clean && babel src --out-dir lib",
```

and the test scripts would look like

```js
"test": "mocha --compilers js:babel-register --require test/init.js test",
"test:watch": "npm run test -- --watch test"
```

The test files would import from `src/`:

```js
import { handleAction, createAction } from '../src';
```